src/pages/bubblegum/stored-nft-data.md

```diff
@@ -4,4 +4,72 @@ metaTitle: Bubblegum - Storing and Indexing NFT Data
 description: Learn more about how NFT data is stored on Bubblegum
 ---
 
-_Coming soon..._
+As mentioned in the [Overview](/bubblegum#read-api), whenever compressed NFTs (cNFTs) are created or modified, the corresponding transactions are recorded on-chain in the ledger, but the cNFT state data is not stored in account space.  This is the reason for the massive cost savings of cNFTs, but for convenience and usability, the cNFT state data is indexed by RPC providers and available via the **the Read API**.
```

Comment:
Should this say "Metaplex Read API" and link to the public repo?

Reply:
Renamed it to "Metaplex Read API" here and where it is introduced in the Overview page (index.md).